navbar.css

```diff
@@ -2,39 +2,22 @@ nav {
 	display: flex;
 	flex-direction: row-reverse;
```

Comment:
I'd take this out and then change the order of the elements within the html document.

Reply:
This is the only one I have issues with, when I change it it messes up, I have tried though!